2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -18,8 +18,88 @@
 */
 //cspell: enable
 
+class DomElement {
+  constructor(tagType, tagId, cssClass, content) {
+    this.tagType = tagType;
+    this.tagId = tagId !== undefined ? tagId : '';
+    this.cssClass = cssClass !== undefined ? cssClass : '';
+    this.content = content !== undefined ? content : '';
+    this.child = [];
+  }
+  addContent(content) {
+    this.content = content;
+  }
+  addChild(child) {
+    if (Array.isArray(child)) {
+      child.forEach((element) => {
```

Comment:
Instead of foreach and push you could use the spread operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax